src/app/pages/user-profile/user-profile.component.html

```diff
@@ -97,10 +85,11 @@ <h6 class="heading-small text-muted mb-4">User information</h6>
                 <div class="col-lg-6">
                 </div>
               </div>
+
               <div class="row">
                 <div class="col-lg-6">
                   <div class="form-group">
-                    <label class="form-control-label" for="input-first-name">First name</label>
```

Comment:
this has nothing to do with adding the on the profile. You will need to pull the name from the auth service and show it here.